MIGRATING.md

```diff
@@ -0,0 +1,305 @@
+# Migrate from the SignalFx Tracing Library for Go
+
+The [Splunk Distribution of OpenTelemetry for Go] replaces the [SignalFx Tracing
```

Comment:
I think it should be "Splunk Distribution of OpenTelemetry Go" rather than "Splunk Distribution of OpenTelemetry for Go" (applies in other places as well)

@theletterf WDYT?